pset-iterators.js

```diff
@@ -115,6 +131,14 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+//  grab list of presidents that were in office between 1850 and 1900
+const presidentsBetween1850and1900 = presidents.filter(presidentObj => {
+  //  presidentObj.took_office.split('-')[0] = year the president took office
+  return (presidentObj.took_office.split('-')[0] > 1850) & (presidentObj.took_office.split('-')[0] < 1900)
+})
+
+//  print to console
+console.log(presidentsBetween1850and1900)
```

Comment:
@sad3xoxo Good Job here using the filter method on this solution and returning the presidents between 1850 and 1900. Good work splitting the` took_office` and returning the value if it is greater than 1850 & less than 1900.

Take a loop at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)